blogposts/2021/deprecating-lsp.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Deprecating Language Server support"
+author: Eric Fritz
+authorUrl: https://eric-fritz.com
+description: "We're dropping support for Language Server-based code intelligence from our product effective on 2021-09-01. Note that this will affect all instances that are not running a private extension registry, not just those upgrading to the current version of Sourcegraph."
+publishDate: 2021-08-10T12:00-00:00
+tags: [blog]
+slug: deprecating-lsp
+heroImage: /blog/optimizing-code-intelligence-commit-graph.png
+socialImage: https://about.sourcegraph.com/blog/optimizing-code-intelligence-commit-graph.png
+published: true
+---
+
+As of September 1st, 2021, we will be [merging a change](https://github.com/sourcegraph/code-intel-extensions/pull/664) to the Sourcegraph language extensions that drops support for querying Language Servers. This method of [code intelligence](https://docs.sourcegraph.com/code_intelligence) has not been actively supported for over a year.
+
+We believe this capability has been unused in the wild for some time. Removing the associated code paths will greatly reduce cognitive clutter around merging search-based and precise code intelligence results, and we'll become more able to iterate quickly to provide a great user experience.

```

Comment:
Do we want to call out something around the enterprise usage of it was very low based on our analytics(and we've helped the remaining 1 or 2 users migrate (like the section further down)).